Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -566,21 +566,32 @@ void GameList::OpenProperties()
   if (!game)
     return;
 
-  PropertiesDialog* properties = new PropertiesDialog(this, *game);
+  auto property_windows = this->findChildren<PropertiesDialog*>();
+  auto it = std::find_if(property_windows.begin(), property_windows.end(),
+                         [&](auto& p) { return p->get_filepath() == game->GetFilePath(); });
+  if (it != property_windows.end())
+  {
+    (*it)->raise();
```

Comment:
```suggestion
    (*it)->raise();
    return;
```
Would eliminate the need for the else-clause and keep the diff noise down. Personal preference maybe, though.